front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/delivery/Delivery.java

```diff
@@ -0,0 +1,94 @@
+package com.netflix.spinnaker.front50.model.delivery;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.netflix.spinnaker.front50.model.Timestamped;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Delivery implements Timestamped {
+  private String id;
+  private String application;
+  private Long updateTs;
+  private Long createTs;
+  private String lastModifiedBy;
+  private List<Map<String,Object>> artifacts;
+  private List<Map<String,Object>> environments;
+  private Map<String,Object> details = new HashMap<>();
+
+  public Delivery() {
+  }
+
+  @Override
+  public String getId() {
+    return id;
+  }
+
+  public void setId(String id) {
+    this.id = id;
+  }
+
+  public String getApplication() {
+    return application;
+  }
+
+  public void setApplication(String application) {
+    this.application = application;
+  }
+
+  @Override
+  public Long getLastModified() {
+    return updateTs;
+  }
+
+  @Override
+  public void setLastModified(Long lastModified) {
+    updateTs = lastModified;
+  }
+
+  @Override
+  public String getLastModifiedBy() {
+    return lastModifiedBy;
+  }
+
+  @Override
+  public void setLastModifiedBy(String lastModifiedBy) {
+    this.lastModifiedBy = lastModifiedBy;
+  }
+
+  public List<Map<String,Object>> getArtifacts() {
```

Comment:
having just been waist deep in the orca artifacts code I wonder if we should use a different name here (`DeliveryArtifacts` or something?? I hate it but I don't know) just to differentiate between Artifacts in triggers/stages, ExpectedArtifacts, BoundArtifacts, etc, etc

Reply:
Changing to `deliveryArtifacts` because I also hate confusion.